content/docs/error-boundaries.md

```diff
@@ -13,6 +13,13 @@ A JavaScript error in a part of the UI shouldn’t break the whole app. To solve
 
 Error boundaries are React components that **catch JavaScript errors anywhere in their child component tree, log those errors, and display a fallback UI** instead of the component tree that crashed. Error boundaries catch errors during rendering, in lifecycle methods, and in constructors of the whole tree below them.
 
+> Note
+> 
+> Error boundaries do **NOT** work in the following scenarios:
+> * Event Handlers (See below ["How About Try / Catch"](#how-about-trycatch) for details about dealing with errors in event handlers)
+> * Asynchronous code
```

Comment:
Could you elaborate on what is meant by this? I worry a reader could interpret this to mean that error boundaries won't work for async React.

Maybe just something like "(eg `setTimeout` or `requestAnimationFrame` callbacks)"

Reply:
I didn't dive into the source code of componentDidCatch and am not sure about this. The original post didn't say anything about async code with error boundaries. Thanks for pointing out.